stories/components/tabs.stories.tsx

```diff
@@ -89,10 +89,9 @@ const ManyTabsStyled = styled(Tabs, {
   height: 100%;
 
   div[role='tablist'] {
-    overflow-x: ${(props) =>
-      props.orientation === 'vertical' ? 'hidden' : 'auto'};
-    overflow-y: ${(props) =>
-      props.orientation === 'vertical' ? 'auto' : 'hidden'};
+    overflow: ${(props) =>
+        props.orientation === 'vertical' ? 'hidden' : 'auto'}
+      ${(props) => (props.orientation === 'vertical' ? 'auto' : 'hidden')};
```

Comment:
```suggestion
    overflow: ${(props) => props.orientation === 'vertical' ? 'hidden auto' : 'auto hidden'};
```

Reply:
If we already use `ts-pattern` on this project, It could be more readable than a ternary.